offline/QA/Jet/photonjetskinematics.cc

```diff
@@ -0,0 +1,184 @@
+//____________________________________________________________________________..
+
+#include "photonjetskinematics.h"
+#include <fun4all/PHTFileServer.h>
+
+#include <fun4all/Fun4AllReturnCodes.h>
+
+#include <phool/PHCompositeNode.h>
+
+#include <TFile.h>
+#include <phool/getClass.h>
+#include "TH1.h"
+#include "TMath.h"
+//#include <jetbase/JetContainer.h>
+
+
+// Tower includes
+#include <calobase/RawCluster.h>
+#include <calobase/RawClusterContainer.h>
+#include <calobase/RawClusterUtility.h>
+#include <calobase/RawTower.h>
+#include <calobase/RawTowerContainer.h>
+#include <calobase/RawTowerGeom.h>
+#include <calobase/RawTowerGeomContainer.h>
+#include <calobase/TowerInfo.h>
+#include <calobase/TowerInfoContainer.h>
+#include <calobase/TowerInfoDefs.h>
+
+// Cluster includes
+#include <calobase/RawCluster.h>
+#include <calobase/RawClusterContainer.h>
+
+
```

Comment:
We'll also need a few standard headers as well...
```suggestion

// c++ includes
#include <algorithm>
#include <cassert>
#include <iostream>
#include <vector>

```